src/classes/BDI_DataImport_API.cls

```diff
@@ -37,6 +37,11 @@
 global with sharing class BDI_DataImport_API {
 
      private class BDIException extends Exception {}
+    
+	public static boolean userCanEditSettings {get {
```

Comment:
Definitely not needed in this class. At first, I was checking everywhere down the chain and deleted a lot of those checks but I guess accidentally left this helper. Should only be used in controller.